src/lib/components/private/QDocsSection.svelte

```diff
@@ -10,16 +10,22 @@
     noCode?: boolean;
   };
 
+  // #region:    --- Props
   let { title, noCode = false, sectionDescription, children }: QDocsSectionProps = $props();
+  // #endregion: --- Props
 
-  const snippets = getContext<undefined | (() => Record<string, string>)>(QDocsCtxName.snippets);
+  // #region:    --- Reactive variables
+  let dialog = $state(false);
 
-  const code = $derived(snippets && snippets()[title]);
+  const ctx = docsCtx.get();
+  // #endregion: --- Reactive variables
 
-  let dialog = $state(false);
+  // #region:    --- Derived values
+  const code = $derived(ctx?.snippets && ctx.snippets[title]);
```

Comment:
```suggestion
  const code = $derived(ctx?.snippets?.[title]);
```
Wouldn't this style work? I feel we have a lot of `&&` where it might not be needed, and it's also not super nice for readability.

Reply:
Yes it can (and will) be changed